ast/ast.go

```diff
@@ -1920,3 +1921,13 @@ type UpdateItem struct {
 	Path []*Ident // len(Path) > 0
 	Expr Expr
 }
+
+// SequenceExpr is sequence expression node.
+//
+// SEQUENCE {{.Name | sql}}
+type SequenceExpr struct {
+	// pos = Sequence
+	// end = Name.end
+	Sequence token.Pos // position of "SEQUENCE" keyword
+	Name     *Ident
```

Comment:
Please add empty lines between pos/end comments and `token.Pos` fields.

```suggestion
	// pos = Sequence
	// end = Name.end

	Sequence token.Pos // position of "SEQUENCE" keyword

	Name     *Ident
```